### PR TITLE
fixed '_x' latex  issue

### PR DESCRIPTION
**Pull request by rijusougata13**
Leading underscores and `latex()`
#### References to other Issues or PRs

this should fix #14718

### CODE REVIEW

File: sympy/printing/latex.py [hunk 1]
```diff
@@ -128,9 +128,10 @@ def latex_escape(s):
     We can't use verbatim easily with mathjax, so escaping is easier.
     Rules from https://tex.stackexchange.com/a/34586/41112.
     """
-    s = s.replace('\\', r'\textbackslash')
+    
     for c in '&%$#_{}':
         s = s.replace(c, '\\' + c)
+    s = s.replace('\\', r'\textbackslash')
```
Comment (eric-wieser) [inline sympy/printing/latex.py:134]: This isn't correct, this means that `_` is escaped to `\textbackslash_` instead of `\_`

**Review by eric-wieser — CHANGES_REQUESTED** [review]
This isn't right - can you give more information about what is going wrong?

**Comment by rijusougata13** [comment]
so can you clarify this one more what do you want instead of what?

**Comment by eric-wieser** [comment]
This doesn't solve the problem in the linked issue, and it breaks an existing test. I don't know what the right solution is yet unfortunately.

**Comment by rijusougata13** [comment]
ok i am trying to solve this

**Comment by rijusougata13** [comment]
so do you want this if a user type  ' _name' .do you want to take this as
'_name' or take this as 'name'

**Pull request closed without merging**